application/templates/_helpers.tpl

```diff
@@ -35,17 +35,39 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Additional common labels
+*/}}
+{{- define "application.additionalLabels" -}}
+{{- if .Values.additionalLabels }}
+{{- range $key, $value := .Values.additionalLabels }}
```

Comment:
Using

`{{ include "application.tplvalues.render" ( dict "value" $value "context" $ ) }}`

would allow templating using the values content and harmonize the way it's done everywhere. Would you like to change it?